README.md

```diff
@@ -80,6 +81,7 @@ the only label value published for features is the string `"true"`._
   "node.alpha.kubernetes-incubator.io/nfd-rdt-<feature-name>": "true",
   "node.alpha.kubernetes-incubator.io/nfd-pstate-<feature-name>": "true",
   "node.alpha.kubernetes-incubator.io/nfd-network-<feature-name>": "true"
+  "node.alpha.kubernetes-incubator.io/nfd-selinux-<feature-name>": "true"
```

Comment:
added line without comma creates bad json syntax here as there are now 2 lines without comma. 
This is README so no actual error however will happen.
Is it also the reason why GH diff display highlights one colon character here with red?